### ISSUE TITLE
Hyperlinks in PlantUML not clickable

### ISSUE THREAD

**Issue by xnd1124** [gregzuro/obsidian-kroki#3]
Hyperlinks are shown but not clickable
Example:
```plantuml
Bob -> Alice : hello [[https://www.google.com]]
Alice -> Wander: hello
Wonderland -> next: hello
next -> Last: hello
Last -> next: hello
next -> Wonderland : hello
Wonderland -> Alice : WTF?
Alice -> Bob: hello
```
![image](https://user-images.githubusercontent.com/6578710/125423702-ee7db48e-10ff-412d-b179-dcdc828b1d25.png)

**Comment by xnd1124** [comment]
In [obsidian-plantuml](https://github.com/joethei/obsidian-plantuml) for hyperlinks you need to use plantuml-map type.

> You can also use plantuml-ascii to generate ASCII Art, or plantuml-map to generate a png image with clickable links.

Can you fix it in this plugin?

**Comment by gregzuro** [comment]
> Hyperlinks are shown but not clickable
> Example:
> 
> ```wsd
> Bob -> Alice : hello [[https://www.google.com]]
> Alice -> Wander: hello
> Wonderland -> next: hello
> next -> Last: hello
> Last -> next: hello
> next -> Wonderland : hello
> Wonderland -> Alice : WTF?
> Alice -> Bob: hello
> ```

For this example, kroki does return svg content with a clickable link: https://kroki.enactive.net/plantuml/svg/eNpzyk9S0LVTcMzJTE5VsFLISM3JyVeIjs4oKSkottLXLy8v10vPz0_PSdVLzs-NjeWCKATqCE_MS0ktgurgCs8H8XKAYiC5vNSKEpgMiA0S80kshouB2LjUIZmE3XCYU8ND3OwRznHKT4IpBwDm50S6

That is rendered by obsidian (?) without the link, however.

**Comment by xnd1124** [comment]
I think problem in tag \<img\> which not allow interactive action.
If I manualy replace it on another tag \<object\> all works fine
![image](https://user-images.githubusercontent.com/6578710/125670359-0b5cfc99-74ba-49ef-92b7-f8d3769438f0.png)
![image](https://user-images.githubusercontent.com/6578710/125670393-61e4959f-49a4-473c-b037-16b54691af72.png)
P.S. tag \<img\> is result of work of svgProcessor
![image](https://user-images.githubusercontent.com/6578710/125670598-0d29dc0d-979b-4a6a-83a3-050689126059.png)

**Comment by gregzuro** [comment]
> I think problem in tag <img> which not allow interactive action.
> If I manualy replace it on another tag <object> all works fine

I've got it working with a variant of this, yeah.

I'll have it up later today.

Thanks!

**Comment by gregzuro** [comment]
Fixed in 5c1fe0a096a774071794204e6b86fbd93a35d345.
